distribution/scripts/jmeter/run-performance-tests.sh

```diff
@@ -121,6 +121,32 @@ declare -A test_scenario6=(
     [skip]=false
 )
 declare -A test_scenario7=(
+    [name]="h2_h2_client_downgrade"
+    [display_name]="HTTP/2 client downgrade service (h2 -> h1)"
+    [description]="An HTTP/2 (over TLS) client, which sends requests to an HTTPS back-end service and gets HTTPS responses."
+    [bal]="h2_h2_passthrough.balx"
+    [bal_flags]=""
+    [path]="/passthrough"
+    [jmx]="http2-post-request.jmx"
+    [protocol]="https"
+    [use_backend]=true
+    [backend_flags]="--ssl --key-store-file $HOME/ballerinaKeystore.p12 --key-store-password ballerina"
+    [skip]=false
+)
+declare -A test_scenario8=(
+    [name]="h2_h2_server_downgrade"
+    [display_name]="HTTP/2 server downgrade service (h1 -> h2)"
+    [description]="An HTTP/2 (over TLS) server, which accepts requests from an HTTPS client and sends HTTPS responses."
```

Comment:
Shall we rephrase the description to "An HTTP/2 (with TLS) server, accepts requests from an HTTP/1.1(with TLS) client which downgrades the connection to HTTP/1.1(with TLS)"